Module "client/java" not testable on its own anymore

**Describe the bug**
Yesterday I was able to run the module mentioned above on it's own via `mvn verify` in the folder. This is not possible anymore since this PR was merged: https://github.com/operaton/operaton/pull/249

**To Reproduce**
Steps to reproduce the behavior:
1. cd to `client/java`
2. Run `mvn verify`
3. Observe the error:

```
Apache Maven 3.9.9 (8e8579a9e76f7d015ee5ec7bfcdc97d260186937)
Maven home: /Users/zoeller/.sdkman/candidates/maven/current
Java version: 17.0.1, vendor: Eclipse Adoptium, runtime: /Users/zoeller/.sdkman/candidates/java/17.0.1-tem
Default locale: de_DE, platform encoding: UTF-8
OS name: "mac os x", version: "14.6.1", arch: "aarch64", family: "mac"
[INFO] Error stacktraces are turned on.
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.operaton.commons:operaton-commons-logging:jar is missing. @ line 53, column 17
[ERROR] 'dependencies.dependency.version' for org.operaton.commons:operaton-commons-typed-values:jar is missing. @ line 58, column 17
[WARNING] 'build.pluginManagement.plugins.plugin.(groupId:artifactId)' must be unique but found duplicate declaration of plugin org.apache.maven.plugins:maven-javadoc-plugin @ org.operaton.bpm:operaton-root:1.0.0-beta-2, /Users/zoeller/sources/operaton/operaton-bpm-platform/pom.xml, line 183, column 17
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.operaton.commons:operaton-commons-logging:jar is missing. @ line 53, column 17
[ERROR] 'dependencies.dependency.version' for org.operaton.commons:operaton-commons-typed-values:jar is missing. @ line 58, column 17
[WARNING] 'build.pluginManagement.plugins.plugin.(groupId:artifactId)' must be unique but found duplicate declaration of plugin org.apache.maven.plugins:maven-javadoc-plugin @ org.operaton.bpm:operaton-root:1.0.0-beta-2, /Users/zoeller/sources/operaton/operaton-bpm-platform/pom.xml, line 183, column 17

    at org.apache.maven.project.DefaultProjectBuilder.build (DefaultProjectBuilder.java:389)
    at org.apache.maven.graph.DefaultGraphBuilder.collectProjects (DefaultGraphBuilder.java:349)
    at org.apache.maven.graph.DefaultGraphBuilder.getProjectsForMavenReactor (DefaultGraphBuilder.java:340)
    at org.apache.maven.graph.DefaultGraphBuilder.build (DefaultGraphBuilder.java:76)
    at org.apache.maven.DefaultMaven.buildGraph (DefaultMaven.java:448)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:197)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
[ERROR]   
[ERROR]   The project org.operaton.bpm:operaton-external-task-client:1.0.0-beta-2 (/Users/zoeller/sources/operaton/operaton-bpm-platform/clients/java/client/pom.xml) has 2 errors
[ERROR]     'dependencies.dependency.version' for org.operaton.commons:operaton-commons-logging:jar is missing. @ line 53, column 17
[ERROR]     'dependencies.dependency.version' for org.operaton.commons:operaton-commons-typed-values:jar is missing. @ line 58, column 17
```

**Expected behavior**
The module should be testable on its own

--- a/clients/java/client/pom.xml
+++ b/clients/java/client/pom.xml
@@ -31,6 +31,13 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
+        <groupId>org.operaton.commons</groupId>
+        <artifactId>operaton-commons-bom</artifactId>
+        <version>${project.version}</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+      <dependency>
         <groupId>org.operaton.bpm</groupId>
         <artifactId>operaton-core-internal-dependencies</artifactId>
         <version>${project.version}</version>

